src/platform_impl/apple/appkit/monitor.rs

```diff
@@ -237,6 +237,21 @@ impl MonitorHandle {
         }
     }
 
+    pub fn bit_depth(&self) -> u16 {
+        run_on_main(|mtm| {
+            match self.ns_screen(mtm) {
+                Some(screen) => unsafe {
+                    (NSBitsPerPixelFromDepth(screen.depth())
+                        * screen
+                            .colorSpace()
+                            .map(|color_space| color_space.numberOfColorComponents())
+                            .unwrap_or(4)) as u16
+                },
```

Comment:
Please help @madsmtm :cold_sweat:.
Not sure why this is unsafe or if there is a better way to do this.
I guess I could also just assume that the number of color components is 4?

Reply:
Resolved by the PR completely changing.